plugins/CoreHome/angularjs/http404check.js

```diff
@@ -7,7 +7,7 @@
 
         function isClientError(rejection)
         {
-            if (rejection.status === 500) {
+            if (rejection.status === 500 || rejection.status === -1) {
```

Comment:
Can we do <= 0 to potentially catch more problems?

Reply:
Done. Although not sure when it would be 0 or lower than that but we can always change it again later if needed for some reason 👍